.changeset/giant-vans-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": minor
+---
+
+Introduces a validation function for the matrix widget (extracted from matrix scoring function).
```

Comment:
I think this might need to be updated :)

Reply:
Gah! I mixed commits from another PR.  I'll clean this PR up.